doc/user/formats.rst

```diff
@@ -130,3 +130,66 @@ fractional_ranges
         be masked if :math:`[\frac{i-0.5}{nchans}, \frac{i+0.5}{nchans}]`
         overlaps any of the ranges. Note that this means that channel 0
         is centred at 0.0 but extends below it.
+
+Primary beam
+------------
+The file contains the information in a compact Fourier-transform ("aperture
+plane") form. It comprises a 3-dimensional array J of Jones matrices. Another
+three arrays :math:`\nu`, :math:`y` and :math:`x` indicate the frequency (in
+Hz) and spatial position (in metres) of each sample along the axes.
+
+To determine the response at a sampled frequency :math:`\nu_k` and some
+direction, turn the direction into direction cosines :math:`l` and :math:`m`
+relative to the pointing centre. Then the Jones
+matrix for the response is
+
+.. math:: \frac{1}{|x|\cdot|y|}
+          \sum_{i,j} e^{-2\pi i (x_i l + y_j m)\nu_i/c} J_{k,j,i}.
+
+Note the reversed axis order in accessing :math:`J`.
```

Comment:
I'd for clarity include: 'where i refers to the horizontal direction of the aperture plane'

Reply:
I've moved the "see ..." closer to this text (it points at https://sdp-jenkins.kat.ac.za/job/ska-sa/job/katsdpmodels/job/primary-beam-file-format/API_20docs/katsdpmodels.html#katsdpmodels.primary_beam.AltAzFrame).